content/en/blog/_posts/2023-08-21-job-update-post.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: blog
+title: "Kubernetes 1.28: New Job Features"
```

Comment:
this title is not very enticing.

How about `Advanced failure handling using Jobs`?

That title would also cover `podFailurePolicy`, which we haven't promoted in a blogpost yet. We could add a quick mention.

Reply:
I'd say “Improved failure handling for Jobs”

Reply:
I like “Improved failure handling for Jobs”, as `backoffLimitPerIndex` and `podRepalcementPolicy: Failed` seem actually as desired behavior in most cases.

However, I think the current "Kubernetes 1.28: New Job Features" title makes also sense as these are the only new Job features that enter alpha in 1.28. But focusing attention also makes sense.

Reply:
Retitled and added a mention for pod failure policy